crates/core/c8y_api/src/json_c8y_deserializer.rs

```diff
@@ -462,7 +462,7 @@ impl From<C8yDownloadConfigFile> for ConfigPayload {
     fn from(value: C8yDownloadConfigFile) -> Self {
         ConfigPayload {
             config_type: value.config_type,
-            remote_url: Some(value.url),
+            remote_url: value.url,
```

Comment:
From what I remember, `remote_url` was left optional as a device profile operation can be triggered locally as well with only a `tedgeUrl` field and no `remoteUrl`.

Reply:
reverted